tests/datalab/test_factory.py

```diff
@@ -23,7 +24,7 @@ class TestIssueManager(IssueManager):
 
     TestIssueManager = register(TestIssueManager, task=None)
 
-    issue_types = Datalab.list_possible_issue_types()
+    issue_types = lab.list_possible_issue_types()
```

Comment:
Reminder to self:

Revisit this line after the issue type listing methods have been refactored.